app/src/main/java/org/mozilla/fenix/wallpapers/WallpaperManager.kt

```diff
@@ -143,6 +145,39 @@ class WallpaperManager(
         }
     }.getOrNull()
 
+    private fun scaleBitmapToBottom(bitmap: Bitmap, view: ImageView) {
```

Comment:
I'm going to trust you on this calculation here.

Reply:
Thanks, but I had some help from [this article](https://medium.com/mobile-app-development-publication/android-image-startcrop-and-endcrop-scaling-18d7297f1e7c). Should I maybe include that in a comment?

Reply:
Sure.  There's any example in `StrictModeManager`